addons/website/static/tests/builder/custom_tab/builder_components/builder_number_input.test.js

```diff
@@ -83,6 +84,31 @@ test("input with classAction and styleAction", async () => {
     });
 });
 
+test("input kept on async action", async () => {
+    const def = new Deferred();
+    addActionOption({
+        customAction: {
+            getValue: ({ editingElement }) => editingElement.dataset.test,
+            apply: async ({ editingElement, value }) => {
+                await def;
+                editingElement.dataset.test = value;
+            },
+        },
+    });
+    addOption({
+        selector: ".test-options-target",
+        template: xml`<BuilderNumberInput action="'customAction'"/>`,
+    });
+    await setupWebsiteBuilder(`<div class="test-options-target" data-test="1">Hello</div>`);
+    await contains(":iframe .test-options-target").click();
+    await contains(".options-container input").edit("2");
+    await click(".options-container input");
+    await fill("3");
```

Comment:
it seems that `await contains(".options-container input").fill("3"); ` triggers a commit